### DISCUSSION TITLE
Only 1  Edge renders. All nodes Render

### DISCUSSION THREAD

**Discussion by prsfreal** [neo4j-contrib/neovis.js#386] [Q&A]
I am new to Neo4j and Neovis. I have a valid instance set up. I tried to use the GOT example. All nodes load but only one edge loads. I have also added my own data to the instance. No matter the query, it always returns all Nodes and just one Edge. The queries and data are valid as the same query works in AuraDb console and Graphology library usage. Not sure what is causing this. I cannot find any documentation referring to this issue.

**Comment by ghost** [comment]
Hey did you figure this out? if yes, please let me know, I am facing the same issue

**Comment by prsfreal** [comment]
This <https://github.com/neo4j-contrib/neovis.js/issues/367> is the issue. TLDR - AuraDB Free instance uses an ID system = Infinite. Neovis cannot handle. You need switch to a local instance or a paid version of AuraDB. Honestly I just switched to D3 and have had great results. Better docs. Less abstraction.

Hope this helps.

**Comment by ghost** [comment]
oh okay Thanks a lot for helping me out.
Just one more doubt in case you know, this d3 js, can I use this for graph visualisation when I am using auraDB provided dataset directly or I have to make local database?
I mean can d3 js use the neo4j instance?

**Comment by prsfreal** [comment]
Yes, you can do that. D3 requires a little more setup but offers great
control in the end. I can send you a code segment later if you want that
will give you a basic layout

**Reply by ghost** [reply]
Oh okay thank you very much
I'll try d3 js then, will ask you for the code segment if I get stuck
Thanks again!

**Comment by shaikh1994** [comment]
I tried the GOT dataset on neo4j desktop and it also shows same one edge. Any idea why is that ?
